src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+
+}
+
+.logo__img {
+  padding: 10px 0;
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  display: block;
+
+}
+
+.nav__list {
+  margin: 0;
+  list-style: none;
+  padding: 0;
+  display: flex;
+
+}
+
+.nav__link {
+  line-height: 60px;
+  height: 60px;
+  display: block;
+  text-decoration: none;
+  color: #000;
+  white-space: nowrap;
+  text-transform: uppercase;
+  position: relative;
+
+}
+
+.header {
+
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+}
+
+.nav__item:not(:first-child) {
+
+  display: block;
+
+  margin-left: 20px;
+}
```

Comment:
1. I like this solution🔥
2. Remove those extra spaces
3. `display: block;` is redundant